<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storage</class>
 <widget class="QWidget" name="storage">
  <property name="windowTitle">
   <string>Zarządzanie miejscami przechowywania</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <property name="horizontalSpacing"><number>10</number></property>
   <property name="verticalSpacing"><number>10</number></property>
   <!-- Storage Form -->
   <item row="0" column="0">
    <layout class="QFormLayout" name="storageForm">
     <item row="0" column="0">
      <widget class="QLabel" name="label"><property name="text"><string>Miejsce przechowywania</string></property></widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <!-- Action Buttons -->
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="buttonsLayout">
     <item><widget class="QPushButton" name="pushButton_add"><property name="text"><string>Dodaj</string></property></widget></item>
     <item><widget class="QPushButton" name="pushButton_edit"><property name="text"><string>Edytuj</string></property></widget></item>
     <item><widget class="QPushButton" name="pushButton_delete"><property name="text"><string>Usuń</string></property></widget></item>
    </layout>
   </item>
   <!-- List View -->
   <item row="0" column="1" rowspan="2">
    <widget class="QListView" name="listView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
     </property>
    </widget>
   </item>
   <!-- Dialog Buttons -->
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation"><enum>Qt::Horizontal</enum></property>
     <property name="standardButtons"><set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>